app/services/s3_query_service.rb

```diff
@@ -99,15 +99,37 @@ def client
     @client ||= Aws::S3::Client.new(region: region, credentials: credentials)
   end
 
+  # required, accepts ETag, Checksum, ObjectParts, StorageClass, ObjectSize
+  def self.object_attributes
+    [
+      "ETag",
+      "Checksum",
+      "ObjectParts",
+      "StorageClass",
+      "ObjectSize"
+    ]
+  end
+
+  def get_s3_object_attributes(key:)
+    response = client.get_object_attributes({
+      bucket: bucket_name,
+      key: key,
+      object_attributes: self.class.object_attributes
+    })
+    response.to_h
+  end
+
   def get_s3_object(key:)
+    object_attributes = get_s3_object_attributes(key: key)
+
     response = client.get_object({
```

Comment:
This was the line I thought should change to get_object_attributes. It is only used here:https://github.com/pulibrary/pdc_describe/blob/7bb589a62ec74204f59431f934d909a08a492c37/app/services/s3_query_service.rb#L116 Which is just utilized for creating an S3File which is all about the attributes not the file data